sql/dj_setup_database.in

```diff
@@ -278,6 +279,11 @@ install-examples)
 	DBUSER=$domjudge_DBUSER PASSWD=$domjudge_PASSWD symfony_console domjudge:load-example-data
 	;;
 
+install-loadtest)
+  read_dbpasswords
```

Comment:
Do we want to prompt the user here to confirm that they took a DB dump before?

Reply:
"Please confirm you took a dump before?"